pkg/cli/mirror/manifests.go

```diff
@@ -261,3 +299,16 @@ func WriteCatalogSource(mapping image.TypedImageMapping, dir string) error {
 	logrus.Infof("Wrote CatalogSource manifests to %s", dir)
 	return nil
 }
+
+// WriteUpdateService will generate an UpdateService object and write it to disk
+func WriteUpdateService(release, graph image.TypedImage, dir string) error {
+	updateService, err := generateUpdateService("update-service", release.Ref, graph.Ref)
```

Comment:
Are we gonna have a problem with the fact we hardcode the name of the UpdateService here which essentially enforce "there will be only one UpdateService and will be overwritten in subsequent runs"?

Reply:
Good question. I think this is probably okay because AFAIK one instance of the `UpdateService` would be run in a cluster at any given time. Since we are resolving the graph image to digest, I would think we would want it overwritten. 

Maybe @wking can shed some light here. Is there a use case where overwriting the UpdateService manifest could cause a problem?

Reply:
it is theoretically possible to run multiple UpdateService instances in the same cluster.  That seems pretty niche though.  As long as you have a name for your UpdateService that's fairly unique to `oc-mirror`, I expect conflicts with other tooling or manually-created UpdateService would be unlikely.  And we also currently have [rbhz#1939855][1], making it harder than it should be to get the update service to notice an UpdateService bump to `graphDataImage`.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1939855

Reply:
@dinhxuanvu I agree. I am good with updating the name on the manifest to something like `update-service-oc-mirror`. 
Also thanks for the info @wking on bumping the `graphDataImage`, it is something we should keep in mind though since we do not actually apply any manifests to the cluster, it would not impact `oc-mirror` functionality. Possibly warrants a warning in docs.